pr_agent/algo/ai_handlers/litellm_ai_handler.py

```diff
@@ -68,6 +70,7 @@ def __init__(self):
             )
         if get_settings().get("AWS.BEDROCK_REGION", None):
             litellm.AmazonAnthropicConfig.max_tokens_to_sample = 2000
+            litellm.AmazonAnthropicClaude3Config.max_tokens = 2000
```

Comment:
In litellm, a separate Config class is created for AmazonAnthropicClaude3

Similar to the following pull request, the value of max_tokens is hardcoded.
https://github.com/Codium-ai/pr-agent/pull/483#discussion_r1407657224

Reply:
Discussion:
Another approach within chat_completion is to handle which configuration to set based on the model, either AmazonAnthropicConfig or AmazonAnthropicClaude3Config.